source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -253,6 +253,75 @@ const callback = filename => (err, data) => {
 };
 ```
 
+#### Async functions
```

Comment:
I think this would be a more descriptive heading for people scanning the page:

```suggestion
#### Use of `async`/`await`
```

Reply:
I like the new heading but it doesn't look like the font for code in headings adds much. What do you think about leaving at the backticks?

With backticks:
![image](https://user-images.githubusercontent.com/87140/86226226-415dd600-bb83-11ea-8c63-37d2d375f677.png)

Without backticks:
![image](https://user-images.githubusercontent.com/87140/86226241-46bb2080-bb83-11ea-8ef1-038d37eebbb8.png)

Reply:
I think without backticks is probably bit better design wise

Reply:
Agreed 👍  thanks both.